manager/manager.go

```diff
@@ -308,6 +311,7 @@ func (m *Manager) Run(parent context.Context) error {
 	api.RegisterRaftMembershipServer(m.server, proxyRaftMembershipAPI)
 	api.RegisterControlServer(m.localserver, localProxyControlAPI)
 	api.RegisterControlServer(m.server, authenticatedControlAPI)
+	api.RegisterResourceAllocatorServer(m.server, authenticatedResourceAPI)
```

Comment:
No raft proxy for this service? Will clients be expected to connect directly to the leader?